helm/vmss-prototype/templates/vmss-prototype.yaml

```diff
@@ -10,6 +10,35 @@
 {{- end -}}
 {{- end -}}
 
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kamino
```

Comment:
I would tie this to something like the job name such that multiple deployments of this do not collide